### PR TITLE
Add HW10

### PR DESCRIPTION
**Pull request by Gritskov-Vitaliy**

### CODE REVIEW

File: HW10/script.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+dir="/home/tms/DevOps_TMS/$1"
+filename1="*.sh"
+filename2="*.png"
+
+find_filename1=`find $dir -type f -iname "$filename1"`
```
Comment (mavrik90) [inline HW10/script.sh]: Do find for both filetypes in one command

File: HW10/script.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+find_filename2=`find $dir -type f -iname "$filename2"`
+find_all=`find $dir` 
+
+echo "Print all $filename1 under $dir"; tput setaf 5
+echo $find_filename1; tput setaf 7
+echo "Print all $filename2 under $dir"; tput setaf 2
+echo $find_filename2; tput setaf 7
+echo "Print all files and folders under $dir" ; tput setaf 3
+echo $find_all
```
Comment (mavrik90) [inline HW10/script.sh]: New line

Comment (Gritskov-Vitaliy) [reply HW10/script.sh]: Done!

File: HW10/script.sh (hunk 1 above)
Comment (Gritskov-Vitaliy) [reply HW10/script.sh]: Done!
image

**Pull request merged**